app/controllers/application_controller.rb

```diff
@@ -10,6 +10,10 @@ def check
     render json: { status: "OK", version: ENV["SHA"], environment: Rails.env }, status: :ok
   end
 
+  def after_sign_in_path_for(user)
+    helpers.profile_dashboard_url(user)
```

Comment:
Probably stored_location_for(user) || helpers.profile_dashboard_url(user) would be better.  stored_location_for(user) is the link that was stored in the session. With it we can send the user back to the page they were on before closing the browser. If there isn't one then we fallback on helpers.profile_dashboard_url(user)

Reply:
Ooh interesting, thanks! I think I'll double check with UX that that's the behaviour we want

Reply:
Ah, I misunderstood what the method did. This sounds good 👍